### DISCUSSION TITLE
Getting Output in JSON/Quill Delta

### DISCUSSION THREAD

**Discussion by dukesx** [outline/rich-markdown-editor#313] [Q&A]
Hi, 

Is there a way that we can get the output in JSON/Quill Delta instead of text markdown ?

**Comment by tommoor** [comment]
Nope – this editor is specifically designed to intake and output markdown. I'd suggest using Remirror, it's very customizable:
https://remirror.io/

**Reply by dukesx** [reply]
Okay, so is it necessary that i use the editor to show the output ?

**Reply by tommoor** [reply]
It's recommended. But with cleanup for the slashes used to encode extra newlines the Markdown should otherwise be compatible with any display.

**Reply by dukesx** [reply]
Okay thank you and one last thing, the only way to get output from editor is using onChange?
